karma.conf.js

```diff
@@ -77,11 +74,17 @@ module.exports = function(config) {
       noInfo: true,
     },
 
+    customLaunchers: {
+      Chrome_travis_ci: {
+        base: 'Chrome',
+        flags: ['--no-sandbox']
+      }
+    },
```

Comment:
Is it possible to create a custom launcher for running Chrome locally in headless mode?  I think you can do that with the --no-startup-window flag.